force_wfmanager/central_pane/plot.py

```diff
@@ -60,9 +61,9 @@ class Plot(HasStrictTraits):
     _plot_data = Instance(ArrayPlotData)
 
     #: Datasource of the plot (used for selection handling)
-    #: Listens to: :attr:`analysis_model.selected_step_index
+    #: Listens to: :attr:`analysis_model.selected_step_indices
     #: <force_wfmanager.central_pane.analysis_model.AnalysisModel.\
```

Comment:
If you're familiar with sphinx/rst and know of any way to shorten these long function references but still have the docs link to it that would be really good! As you can see I couldn't figure out a way to do it haha

Reply:
No idea I'm afraid! Google wasn't too helpful either.

Reply:
You can put as much of the tail as you need to be unambiguous, starting with a `.`, (and using `~` in front makes names prettier in actual docs, so
```
:attr:`~.selected_step_indices`
```
will probably work.  If there is ambiguity then Sphinx will raise a warning.